ui/app/src/Views/NavBar/Views.elm

```diff
@@ -9,12 +9,27 @@ import Views.NavBar.Types exposing (Tab, alertsTab, silencesTab, statusTab, none
 navBar : Route -> Html msg
 navBar currentRoute =
     header
-        [ class "navbar navbar-toggleable-md navbar-light "
-        , style [ ( "margin-bottom", "3rem" ), ( "border-bottom", "1px solid rgba(0,0,0,.15)" ) ]
+        [ class "navbar navbar-toggleable-md navbar-light bg-faded mb-5 pt-3 pb-3"
+        , style [ ( "border-bottom", "1px solid rgba(0, 0, 0, .125)" ) ]
         ]
         [ nav [ class "container" ]
-            [ a [ class "navbar-brand", href "#" ] [ text "AlertManager" ]
+            [ a [ class "navbar-brand", href "#" ] [ text "Alertmanager" ]
             , ul [ class "navbar-nav" ] (navBarItems currentRoute)
+            , case currentRoute of
+                SilenceFormEditRoute _ ->
+                    text ""
+
+                SilenceFormNewRoute _ ->
+                    text ""
+
+                _ ->
```

Comment:
To put this up for discussion: I would not render the "New Silence" button on the AlertList page, but only on the SilenceList page. This way interacting with silences is scoped to the silence related views. What do you think?